addons/hr_work_entry/models/hr_work_entry.py

```diff
@@ -87,6 +90,7 @@ def _mark_conflicting_work_entries(self, start, stop):
         """
         Set `state` to `conflict` for overlapping work entries
         between two dates.
+        If `self.ids` is truthy then check conflicts with the corresponding work entries.
```

Comment:
Hello @aj-fuentes 

Despite the fact that the method is not ``api.model`` anymore, and should be converted into an ``api.multi``, the fix looks good to me. 

Thanks a lot for your work !

Reply:
Decorator removed, thanks!